typescript/submissionEvents.d.ts

```diff
@@ -552,6 +558,7 @@ export type BaseFormSubmissionProcessing = {
   key: string
   isDraft: boolean
   submissionTimestamp: string
+  completionTimestamp?: string
```

Comment:
This is on the submission data, we should not need to add it to this type. If we are adding here, we will need to also set this in the API when replaying submission events.

Reply:
Yeah we don't.  Just seemed cleaner, but ok.